.travis.yml

```diff
@@ -10,11 +13,33 @@ cache:
   directories:
     - $HOME/.gradle/caches/
     - $HOME/.gradle/wrapper/
-    
-script:
-  - wget https://raw.githubusercontent.com/sormuras/bach/76983e2f99497738b6f3e9f86bd42fd3a759a275/install-jdk.sh -O $TRAVIS_BUILD_DIR/install-jdk.sh
-  - bash $TRAVIS_BUILD_DIR/install-jdk.sh --feature 11 -L GPL --target $HOME/openjdk11
-  - ./gradlew build codeCoverageReport -PsystemTestJavaHomes="$JAVA_HOME:$HOME/openjdk11"
-after_success:
-  # upload coverage report
-  - bash <(curl -s https://codecov.io/bash)
\ No newline at end of file
+
+install:
+  - ./gradlew build :inspectit-oce-agent:systemTest
+
+jobs:
+  include:
+    - stage: deploy
+      name: "Code Coverage Reporting"
+      jdk: oraclejdk8
+      install: ./gradlew build codeCoverageReport
+      after_success:
+        # upload coverage report
+        - bash <(curl -s https://codecov.io/bash)
+    - name: "Documentation Deployment"
+      jdk: oraclejdk8
+      install: ./gradlew :inspectit-oce-documentation:asciidoctor
+      deploy:
+        provider: s3
+        access_key_id: $AWS_ACCESS_KEY_ID
+        secret_access_key: $AWS_SECRET_ACCESS_KEY
+        bucket: "docs.inspectit.rocks"
+        region: eu-west-1
+        skip_cleanup: true
+        local_dir: inspectit-oce-documentation/build/asciidoc/html5
```

Comment:
if you are not using the zipped version that is released, maybe you can remove this part from the build file of the documentation as zipping has no meaning.. I added this because of the upload, but was not aware we would be able to simply point to the complete dir..

Reply:
Actually, I just realized that with this PR we would have only one documentation available and we would like to have versioned documentation. What does this mean:

1. I want to have documentation for each released version, this can be achieved by having the folder with the version name `.../0.1.M1` (similar to Spring documentation)
2. In addition, we should have a `.../current` folder that shows the documentation for the latest release. This would be used as the link to the official documentation and hopefully index by google as the first choice.

This is why in the ZIP task I added a parent folder with the version name so that unpacking would solve the point 1 above.

Reply:
Removed zip task

Reply:
And a documentation will be kept for each released version

Reply:
OK I kind of changed my mind. Let's keep the zip task and upload the documentation zip also as the release artifact... I really like how it is now in the first release, that Jonas uploaded the agent jar and documentation zip.. That's cool imo, that we also provide this irrelevant of our docs website..